statsmodels/tools/tests/test_sequences.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+from statsmodels.tools import sequences
+
+
+def test_discrepancy():
+    space_1 = [[1, 3], [2, 6], [3, 2], [4, 5], [5, 1], [6, 4]]
+    space_2 = [[1, 5], [2, 4], [3, 3], [4, 2], [5, 1], [6, 6]]
+
+    corners = np.array([[0.5, 0.5], [6.5, 6.5]])
+
+    # From Fang et al. Design and modeling for computer experiments, 2006
+    assert sequences.discrepancy(space_1, corners) == pytest.approx(0.0081, abs=0.0001)
+    assert sequences.discrepancy(space_2, corners) == pytest.approx(0.0105, abs=0.0001)
+
+
+def test_van_der_corput():
+    sample = sequences.van_der_corput(10)
+    out = [0., 0.5, 0.25, 0.75, 0.125, 0.625, 0.375, 0.875, 0.0625, 0.5625]
+    npt.assert_almost_equal(sample, out)
+
+
+def test_primes():
+    primes = sequences.primes_from_2_to(50)
+    out = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
+    npt.assert_almost_equal(primes, out)
```

Comment:
integers can be compared with assert_equal

in general assert_allclose is better than assert_almost_equal because it allows more fine tuned control over atol and rtol

Reply:
Done